app/javascript/onboarding/components/ProfileForm/TextInput.jsx

```diff
@@ -26,7 +26,13 @@ import { FormField } from '@crayons';
 
 export function TextInput(props) {
   const { onFieldChange } = props;
-  const { attribute_name, placeholder_text, description, label } = props.field;
```

Comment:
refactor (nitpick): `field` can be destructured from `props`.

```suggestion
  const { onFieldChange, field } = props;
  const {
    attribute_name,
    placeholder_text,
    default_value,
    description,
    label,
  } = field;
```